frontend/src/pages/cohorts/CohortsPage.tsx

```diff
@@ -18,9 +20,83 @@ import { PageHeader } from "../../shared/components/PageHeader";
 
 function cohortFilterWhereVariables(parsedSearchVals: string[]): CohortWhere[] {
   if (parsedSearchVals.length > 1) {
-    return [{ cohortId_IN: parsedSearchVals }];
+    return [
+      { cohortId_IN: parsedSearchVals },
+      {
+        hasCohortCompleteCohortCompletes_SOME: {
+          type_IN: parsedSearchVals,
+        },
+      },
+      {
+        hasCohortCompleteCohortCompletes_SOME: {
+          endUsers_INCLUDES: parsedSearchVals[0],
+        },
+      },
+      {
+        hasCohortCompleteCohortCompletes_SOME: {
+          pmUsers_INCLUDES: parsedSearchVals[0],
+        },
```

Comment:
Note here how the `_INCLUDES` suffix was used with these two fields when the user performs bulk search or single-value search instead of `_IN` or `_CONTAINS`, respectively.

Unlike other fields which have a data type of string, these fields have a data type of an array of strings. The only filter parameter available for this data type is `_INCLUDES`, which is why I'm using it here.

To enforce the same fuzzy searching logic for these fields as we do for other fields, we'll likely have to write a resolver to handle querying these fields, which I recommend saving as a separate effort/ticket.